### DISCUSSION TITLE
Installation on remote Host, can't get the hub to work.

### DISCUSSION THREAD

**Discussion by Zmove** [getcandy/hub#65] [Help]
Hello,

I'm quite new to laravel and nuxtjs and I'm completely lost about getcandy installation.

I followed documentation, and used the api boilerplate to get the api working.

First, here is my folder hierarchy : 
- getcandy.mywebsite.com
    - api (laravel is installed here, so my document root is getcandy.mywebsite.com/api/public)
    - hub (hub is installed here)

If I go to url : https://getcandy.mywebsite.com/api/v1, I can see this, so I think the API is working.
![image](https://user-images.githubusercontent.com/1314678/138588892-6e056039-355e-4b17-8494-c840ea354379.png)

And Here is my .env file for the laravel application
![image](https://user-images.githubusercontent.com/1314678/138588729-c35ad3f6-cff9-46bb-9889-63b2eb2c18bd.png)

Here is my .env file for the hub application
![image](https://user-images.githubusercontent.com/1314678/138588806-7842549f-1437-4405-b28a-57ed9f7de452.png)

From here, I don't know what to do. I would believe that the login page would be accessible from getcandy.mywebsite.com/login, but I get a 404 error whereas I used the api boilerplate github repository that define a /login route.

I'm stuck here, could you please help me ?

Thank you

**Comment by alecritson** [comment]
Hi @Zmove 

So the hub needs a node server to run, if you have this in a sub folder then you would need to make sure that you start a node server in the `hub` folder. In Nuxt I believe you can do this via the command `yarn start` but would mean you'd have to keep the terminal session alive, so would need to be started and managed in the background to keep it alive.

From there, you should be able to get access to the hub via `getcandy.mywebsite.com/hub`
